maximbircu/devtools-library
# Improve EnumTool UX

Closes: #35 #43 


### What has been done
 1. Add a different EnumTool UX for large options size.
 2. Fix crash when selecting twice the same option 

### Screenshots
Android | iOS
:-: | :-:
<img src="https://user-images.githubusercontent.com/12527390/80121894-810dad80-8595-11ea-83db-911cd600d306.png" width=200/>| iOS view will be added in #26 |

### How to test
Usecase1
1. Go to 'whatever screen where enum tools are available "MEMORY" for example'
1. Click on 'second-option'
1. Then click one more time on 'second-option'
1. App will force-stop

Usecase2
1. Open the sample app;
1. Find the Enum tool [Custom Provider];
1. Make sure you can update its configuration value and the configuration value is persisted.
